src/main/scala/no/ndla/learningpathapi/service/ReadService.scala

```diff
@@ -42,20 +47,20 @@ trait ReadService {
     def withIdV2(learningPathId: Long,
                  language: String,
                  fallback: Boolean,
-                 user: UserInfo = UserInfo.get): Option[LearningPathV2] = {
+                 user: UserInfo = UserInfo.get): Try[LearningPathV2] = {
```

Comment:
Veldig bra med Try i stedet for Option. Gjør koden lettere å lese i dei andre servicene.